easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -201,10 +202,17 @@ def configure_step(self, srcdir=None, builddir=None):
 
         if '-DCMAKE_BUILD_TYPE=' in self.cfg['configopts']:
             if self.cfg.get('build_type') is not None:
-                self.log.warning('CMAKE_BUILD_TYPE is set in configopts. Ignoring build_type')
+                self.log.warning("CMAKE_BUILD_TYPE is set in configopts. Ignoring 'build_type' easyconfig parameter.")
         else:
             options['CMAKE_BUILD_TYPE'] = self.build_type
 
+        if '-DCMAKE_INSTALL_LIBDIR=' in self.cfg['configopts']:
+            self.log.warning(
```

Comment:
I would make this `info`, not warning, we don't need to "scare" people when `CMAKE_INSTALL_LIBDIR` is used in `configopts`?

Reply:
Shall I also change the warning for `CMAKE_BUILD_TYPE`?

Reply:
fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/3227/commits/805eae2e9457546b357828d8069056ebde4a805a